site/assets/js/application.js

```diff
@@ -169,4 +169,24 @@
     icon: '#'
   }
   anchors.add('.bd-content > h2, .bd-content > h3, .bd-content > h4, .bd-content > h5')
+
+  // Open in StackBlitz logic
+  document.querySelectorAll('.btn-edit')
+    .forEach(function (btn) {
+      var tooltipBtn = new bootstrap.Tooltip(btn)
+
+      btn.addEventListener('mouseleave', function () {
+        // Explicitly hide tooltip, since after clicking it remains
+        // focused (as it's a button), so tooltip would otherwise
+        // remain visible until focus is moved away
+        tooltipBtn.hide()
+      })
+
+      btn.addEventListener('click', function (e) {
+        var htmlSnippet = e.target.parentNode.nextElementSibling.nextElementSibling.textContent
```

Comment:
Just a note

```suggestion
        var htmlSnippet = e.target.parentNode.previousElementSibling.textContent
```

 may work too.

Reply:
I changed it to `event.target.closest('.bd-content').querySelector('.bd-example').innerHTML`. More readable, with the only issue being it not working on IE.

Is that a concern?

Reply:
IE is not supported any more ;)

Reply:
🥳🥳🥳🥳